### PR TITLE
wishlist and cart page of ecommerce project

### PR DESCRIPTION
**Pull request by devAmlan**
These are the cart page and wish list page of the e-commerce project. Please review my PR.

### CODE REVIEW

**Review by bossmonkey98 — APPROVED** [review]

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
+.cartpage h2{
+    margin:3rem 1rem; 
+    font-size:2.4rem;
+    text-align:center;
+}
+    .cart_products{
```
Comment (humzahasan) [inline pages/cart/cart.css:6]: Suggestion : Try adding a generic class( utility class) in your component library for commonly used flex property and use it in other projects.

File: pages/cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        </nav>
+        </div>
+
+    <div class="cartpage">
+        <h2>MY CART (1)</h2>
+        <div class="cart_products">
+    
+        <div class="added_product">
+        <div class="added_product_image">
+        <img src="../../assets/GMK Kaiju.jpg" alt="">
```
Comment (humzahasan) [inline pages/cart/cart.html:40]: Suggestion: give an alt for all images

**Review by humzahasan — APPROVED** [review]
Code looks good. DO check the suggestions added and implement the same in future iteration.

File: pages/cart/cart.html (hunk 2 above)
Comment (devAmlan) [reply pages/cart/cart.html:40]: will give alt , thanks humza.

File: pages/cart/cart.css (hunk 1 above)
Comment (devAmlan) [reply pages/cart/cart.css:6]: ok I will use generic class from next,  thanks  for the suggestion .

**Pull request merged**